--- lib/python2.7/grp.py.OS2	2017-04-20 21:36:09.802412374 +0200
+++ lib/python2.7/grp.py	2017-04-20 21:38:35.483691613 +0200
@@ -31,6 +31,7 @@
   - ${ETC_GROUP}              (or %ETC_GROUP%)
   - ${ETC}/group              (or %ETC%/group)
   - ${PYTHONHOME}/Etc/group   (or %PYTHONHOME%/Etc/group)
+  - `sys.prefix`/Etc/group
 
 Classes
 -------
@@ -55,6 +56,7 @@
 """
 
 import os
+import sys
 
 # try and find the group file
 __group_path = []
@@ -64,6 +66,7 @@
     __group_path.append('%s/group' % os.environ['ETC'])
 if os.environ.has_key('PYTHONHOME'):
     __group_path.append('%s/Etc/group' % os.environ['PYTHONHOME'])
+__group_path.append('%s/Etc/group' % sys.prefix)
 
 group_file = None
 for __i in __group_path:
